### PR TITLE
XWIKI-13858: Inconsistency between Long Text and Content fields display in AWM wizard and entries

### PR DESCRIPTION
**Pull request by Sereza7**
# Jira URL

https://jira.xwiki.org/browse/XWIKI-13858

# Changes

## Description

* In all places where we can edit this object, we follow the userPreference regarding text editors.

## Clarifications

* The reason for the change we want to revert for [XWIKI-8734](https://jira.xwiki.org/browse/XWIKI-8734) was that the CKEditor was too heavy to get in the UI. However, it seems like nowadays we have it in other places in the UI and it does not pose any issue. 

# Screenshots & Video

Screenshots only for the class editor. On the ticket there's already screenshots showing that the issue has been solved for the entry editor.
Before the PR
With prefered editor = text
Screenshot from 2024-10-24 15-03-24
With prefered editor = Wysiwyg
Screenshot from 2024-10-24 15-03-38
After the PR
With prefered editor = text
Screenshot from 2024-10-24 15-28-02
With prefered editor = Wysiwyg
Screenshot from 2024-10-24 15-28-16

# Executed Tests

Manual tests (see above).
There was no test change when merging https://github.com/xwiki/xwiki-platform/commit/e6b6769e0b582ff79a1cf6c3e0d4ad25a37b34eb ,which is partly what we revert.
# Expected merging strategy

* Prefers squash: Yes 
* Backport on branches:
  * 16.4.X
  * 15.10.X

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml [hunk 1]
```diff
@@ -44,14 +44,13 @@
     #set ($name = 'templateContent')
     #set ($editedDocument = $xwiki.getDocument("$stringtool.removeEnd($className, 'Class')Template"))
     ## Don't load the WYSIWYG editor when editing the class, because it's too heavy.
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml:46]:
```suggestion
```

Doesn't make sense to keep the comment.

File: xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml [hunk 2]
```diff
@@ -44,14 +44,13 @@
...
-    #set ($useWysiwygEditor = false)
   #else
     ## We are editing an application entry so the content must be read from / written to the current document.
     #set ($name = 'content')
     #set ($editedDocument = $tdoc)
     ## Use the preferred content editor.
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml:51]: The comment needs to be moved also (or removed if you think the code is obvious).

**Review by mflorea — CHANGES_REQUESTED** [review]
The comments should be deleted / moved along with the related code.

File: xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml (hunk 1 above)
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml]: Addressed in https://github.com/xwiki/xwiki-platform/pull/3594/commits/4371e521d0183839094f7947c7ca8ef58c649620 :+1:

File: xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml (hunk 2 above)
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml]: Addressed in https://github.com/xwiki/xwiki-platform/pull/3594/commits/4371e521d0183839094f7947c7ca8ef58c649620 :+1: 
I moved the comment around. It doesn't hurt to keep it here as long as it's valid :)

**Comment by Sereza7** [comment]
The comments have been reviewed and the PR is ready for further reviewing or a merge.

Thank you for the review MFlorea!

**Review by mflorea — APPROVED** [review]

**Pull request merged**